artemis-server/src/main/java/org/apache/activemq/artemis/core/management/impl/ActiveMQServerControlImpl.java

```diff
@@ -3796,19 +3796,58 @@ public void createBridge(final String name,
                             final boolean ha,
                             final String user,
                             final String password) throws Exception {
+      createBridge(name,
+              queueName,
+              forwardingAddress,
+              filterString,
+              transformerClassName,
+              retryInterval,
+              retryIntervalMultiplier,
+              initialConnectAttempts,
+              reconnectAttempts,
+              useDuplicateDetection,
+              confirmationWindowSize,
+              clientFailureCheckPeriod,
+              staticConnectorsOrDiscoveryGroup,
+              useDiscoveryGroup,
+              ha,
+              user,
+              password,
+              ActiveMQClient.DEFAULT_CALL_TIMEOUT);
```

Comment:
I would suggest this isnt defaulted here. 

Should keep original behaviour of not setting it in the configuration, and in new method setting it.

Reply:
Resolved, thanks for input!